console/src/components/Services/RemoteSchema/Permissions/Permissions.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useEffect, useState } from 'react';
+import Helmet from 'react-helmet';
+import * as GQL from 'graphql';
+import PermissionsTable from './PermissionsTable';
+import PermissionEditor from './PermissionEditor';
+import { getConfirmation } from '../../../Common/utils/jsUtils';
+import { useIntrospectionSchemaRemote } from '../graphqlUtils';
+import globals from '../../../../Globals';
+import Button from '../../../Common/Button/Button';
+import styles from '../../../Common/Permissions/PermissionStyles.scss';
+import { getTree, getType } from './utils';
+import { DatasourceObject } from './types';
+
+const BulkSelectSection = ({ bulkSelect, permRemoveMultipleRoles }: any) => {
```

Comment:
Add props type

Reply:
Also, can we rename `bulkSelect`? It kinda sounds like a function, but it's a list.